webpack.config.js

```diff
@@ -1,6 +1,13 @@
 var path = require('path')
 const CopyPlugin = require('copy-webpack-plugin')
 
+const standalone = /:standalone$/.test(process.env.NODE_ENV)
+const filesToCopy = ['manifest.konnector', 'package.json', 'README.md', 'LICENSE']
+
+if (standalone) {
+  filesToCopy.push('konnector-dev-config.json')
```

Comment:
why ?

Reply:
Needed by the konnector to be run locally. Using es2015 implies to build the konnector somewhere.

Reply:
Sorry I was not clear. It seems to me that konnector-dev-config is used while `developing` to provide the right data to the konnector for the account creation or account retrieval. When it is used in `production`, we do not need konnector-dev-config since the data is passed by the stack. This is why I do not understand why you include `konnector-dev-config.json` in the build.

Reply:
I think that the `standalone` variable is `true` only when running the konnector with `yarn standalone` but I may be wrong.

Reply:
I do not understand why you need to push it manually. If it is in standalone mode, the file that needs it should require it and it should be included in the build automatically. The developer of a konnector should not need to do this.

Reply:
I need to check but as far as I remember, the standalone mode is just `index.js` run with a specific ENV variable, so there is no place to include `konnector-dev-config.json`.